docs/source/release_notes/release-notes.rst

```diff
@@ -84,33 +91,103 @@ develop
            This is strictly more permissive, but may affect developers that use ``ServerListConfig`` directly, especially if it is being serialized.
            (`Pull Request 3 <https://github.com/palantir/atlasdb/pull/2647>`__)
 
-    *    - |fixed| |metrics|
-         - ``MetricsManager`` now logs failures to register metrics at ``WARN`` instead of ``ERROR``, as failure to do so is not necessarily a systemic failure.
-           Also, we now log the name of the metric as a Safe argument (previously it was logged as Unsafe).
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2636>`__)
+    *    - |improved| |logs|
+         - kvs-slow-log was added on all Cassandra calls. As with the original kvs-slow-log logs, the added logs have the ``kvs-slow-log`` origin.
+           To see the exact log messages, check the ``ProfilingCassandraClient`` class.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2673>`__)
+
+    *    - |new| |metrics|
+         - Metrics were added on all Cassandra calls. The ``CassandraClient`` interface was Tritium instrumented. The following metrics were added:
+
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.multiget_slice
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.get_range_slices
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.batch_mutate
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.get
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.cas
+              - com.palantir.atlasdb.keyvalue.cassandra.CassandraClient.execute_cql3_query
+
+           Note that the table calls mainly use the first three metrics of the above list.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2673>`__)
+
+    *    - |new| |metrics|
+         - Metrics recording the number of Cassandra requests, and the amount of bytes read and written from and to Cassandra were added. The metrics are:
+
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.numReadRequests
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.numWriteRequests
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.bytesRead
+              - com.palantir.atlasdb.keyvalue.cassandra.QosMetrics.bytesWritten
+
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2679>`__)
+
+    *    - |new| |metrics|
+         - Added metrics for cells read.
+           The read cells can be post-filtered at the CassandraKVS layer, when its value is not the latest. In this case, we have the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.CellFilterMetrics.notLatestVisibleValueCellFilterCount
+
+           The cells returned from the KVS layer are then recorded at the metric:
+
+              - com.palantir.atlasdb.AtlasDbMetricNames.numCellsRead
```

Comment:
I think this one comes from SnapshotTransaction.. etc and so on for all the metric names.